packages/quantic/force-app/main/default/lwc/quanticNumericFacet/e2e/quanticNumericFacet.e2e.ts

```diff
@@ -0,0 +1,139 @@
+            facetField: field,
+            facetValue: `${values[selectedIndex].start}..${values[selectedIndex].end}`,
+          });
+        searchResponsePromise = baseFacet.waitForSearchResponse();
+
+        await facet.clickOnFacetValue(selectedIndex);
+
+        await facetDeselectUaRequest;
+        searchResponse = await searchResponsePromise;
+        const {analytics: analyticsForFacetDeselect} =
+          baseFacet.extractDataFromResponse(searchResponse);
+        expect(analyticsForFacetDeselect.actionCause).toEqual('facetDeselect');
+      });
+    });
+
+    test.describe('when selecting a facet value and clicking the clear filter button', () => {
+      test('should trigger a new search and log the corresponding UA analytics events', async ({
+        baseFacet,
+        facet,
+      }) => {
+        const selectedIndex = 0;
+        const {facetId, field, values} = facetData;
+        const facetSelectUaRequest = baseFacet.waitForFacetSelectUaAnalytics({
+          facetId,
+          facetField: field,
+          facetValue: `${values[selectedIndex].start}..${values[selectedIndex].end}`,
+        });
+        let searchResponsePromise = baseFacet.waitForSearchResponse();
+
+        await facet.clickOnFacetValue(selectedIndex);
+
+        await facetSelectUaRequest;
+        let searchResponse = await searchResponsePromise;
+        const {analytics: analyticsForFacetSelect} =
+          baseFacet.extractDataFromResponse(searchResponse);
+        expect(analyticsForFacetSelect.actionCause).toEqual('facetSelect');
+
+        const clearAllUaRequest = baseFacet.waitForFacetClearAllUaAnalytics({
+          facetId,
+          facetField: field,
+        });
+        searchResponsePromise = baseFacet.waitForSearchResponse();
+
+        await facet.clickOnClearSelectionButton();
+
+        await clearAllUaRequest;
+        searchResponse = await searchResponsePromise;
+        const {analytics: analyticsForFacetDeselect} =
+          baseFacet.extractDataFromResponse(searchResponse);
+        expect(analyticsForFacetDeselect.actionCause).toEqual('facetClearAll');
+      });
+    });
+
+    test.describe('when typing in the facet search box input', () => {
+      test('should fetch facet values according to the query typed', async ({
+        baseFacet,
+        facet,
+      }) => {
+        const exampleMin = '1';
+        const exampleMax = '2';
+        const {facetId, field} = facetData;
+        const facetSelectUaRequest = baseFacet.waitForFacetSelectUaAnalytics({
+          facetId: `${facetId}_input`,
+          facetField: field,
+          facetValue: `${exampleMin}..${exampleMax}`,
+        });
+        let searchResponsePromise = baseFacet.waitForSearchResponse();
+
+        await facet.fillFilterMinInput(exampleMin);
+        await facet.fillFilterMaxInput(exampleMax);
+        await facet.clickOnFilterApplyButton();
+
+        await facetSelectUaRequest;
+        let searchResponse = await searchResponsePromise;
```

Comment:
Here too, we should make sure the search request contains at least the min/max value we've just set.

Reply:
good point indeed! added checks!

Reply:
very nice